_posts/2023-05-11-Discontinuing-custom-Logstash-distribution.md

```diff
@@ -7,7 +7,7 @@ date: 2023-05-11 12:30:00 -0500
 meta_description: The OpenSearch Project will continue to provide Logstash plugins while discontinuing the custom distribution.
 ---
 
-One of the core [OpenSearch Project principles](https://opensearch.org/about.html) is to be used everywhere.
+One of the core [OpenSearch Project principles](https://opensearch.org/about.html) includes being used everywhere: in businesses, software, and projects.
```

Comment:
"One of the core OpenSearch Project principles is that OpenSearch can be used everywhere: in businesses, software, and projects."